### DISCUSSION TITLE
Pushing protected branch to Github

### DISCUSSION THREAD

**Discussion by efbe** [six7/figma-tokens#1253] [Ideas]
Hello,

When I turn on the "Require a pull request before merging" along with "Do not allow bypassing the above settings" under the "Protect matching branches" in "Branch protection rules" section for Branch settings on Github to disable direct commits to a protected branch like 'master' I can still push from the plugin UI which is fine, but the result is an infinite loading screen about "Pushing to Github" (see attached image).

<img width="525" alt="protected-branch-push-2" src="https://user-images.githubusercontent.com/777653/188581203-8d2ae811-7730-4bff-8f6a-421abcb18df3.png">

If it's possible could you add some error handling to this screen, please.

Thank you in advance.

**Comment by six7** [comment]
Thanks! I'll create an issue from this.
